packages/overture-schema-divisions-theme/src/overture/schema/divisions/division/models.py

```diff
@@ -0,0 +1,186 @@
+"""Division models for Overture Maps divisions theme."""
+
+from typing import Annotated, Literal
+
+from pydantic import ConfigDict, Field
+
+from overture.schema.core import (
+    OvertureFeature,
+)
+from overture.schema.core.enums import Side
+from overture.schema.core.geometry import Geometry, GeometryTypeConstraint
+from overture.schema.core.models import (
+    CartographicallyHinted,
+    Named,
+    Names,
+    Perspectives,
+    StrictBaseModel,
+)
+from overture.schema.core.types import (
+    CommonNames,
+    CountryCode,
+    Id,
+    RegionCode,
+    WikidataId,
+)
+from overture.schema.divisions.models import CapitalOfDivisionItem
+from overture.schema.divisions.types import Hierarchy
+from overture.schema.validation import (
+    UniqueItemsConstraint,
+)
+
+from ..enums import DivisionClass, PlaceType
+from ..validation import parent_division_required_unless
+
+
+class Norms(StrictBaseModel):
+    """Local norms and standards."""
+
+    # Optional
+
+    driving_side: Annotated[
+        Side | None,
+        Field(
+            description="Side of the road on which vehicles drive in the division.",
+        ),
+    ] = None
+
+
+@parent_division_required_unless("subtype", PlaceType.COUNTRY)
+class Division(OvertureFeature, Named, CartographicallyHinted):
+    """Divisions are recognized official or non-official organizations of people as seen from a given political perspective. Examples include countries, provinces, cities, towns, neighborhoods, etc."""
```

Comment:
This seems to be going over the 88-column limit set in the root `pyproject.toml`. I assume this is because the auto-formatter doesn't know how to break up long docstrings, but maybe we should.

Reply:
https://github.com/mojodna/schema/commit/f78767e5c1bb2b79ce1bea745d9cb7fedac12bb4